<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pushAlarm">
         <property name="text">
          <string>Нажать, когда всё плохо</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="langBox">
         <item>
          <property name="text">
           <string>Немецкий</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Французский</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Арабский</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Китайский</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Белорусский</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Иврит</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startBtn">
         <property name="text">
          <string>Старт/стоп</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="startLabel">
         <property name="text">
          <string>Секунды</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="timerBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Слов в минуту</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="wordsBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="aimTextEdit"/>
       </item>
       <item>
        <widget class="QTextEdit" name="checkTextEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
